### DISCUSSION TITLE
UWP app Deployment?

### DISCUSSION THREAD

**Discussion by MohdShabaz** [unoplatform/uno#10496] [Q&A]
Hi,

We have developed a Universal Window Platform (UWP) application. We have integrated it with the Uno platform to have cross-platform development.

How do we distribute the application?

For now, we have only developed the windows version (desktop & laptop) and aim to distribute it to windows users.
What's the best way to deploy it?

Any help is appreciated.

Thank you,
Regards,
Mohammed Shabaz

**Comment by jeromelaban** [comment]
Deployment for UWP applications is done through the official techniques presented here: https://learn.microsoft.com/en-us/windows/uwp/packaging/
